web_src/less/_base.less

```
@@ -238,6 +238,14 @@ a.muted:hover,
   border-color: var(--color-primary) !important;
 }
 
+.ui.action.input:not([class*="left action"]) > .ui.dropdown.selection {
```

```suggestion
/* dropdown on repo search and explore code */
.ui.action.input:not([class*="left action"]) > .ui.dropdown.selection {
```

Should really comment these.

---

This rule should actually be generalized and applied whenever a `<div>` of class "ui dropdown selection" appears inside action input container to make it behave like `<input>`.

---

Yes, but if we notice this causing issues further down the road, we at least know what it's supposed to be targeting. Of course if you can fabricate a more specific selector, that'd also be an option.